CHANGELOG.md

```diff
@@ -8,7 +8,7 @@ The old Realm Cloud legacy APIs have undergone significant refactoring. The new
 * None.
 
 ### Enhancements
-* None.
+* [RealmApp] RealmLists can now be marked final.
```

Comment:
```suggestion
* RealmLists can now be marked final. (Issue [#6892](https://github.com/realm/realm-java/issues/6892))
```

The [RealmApp] prefix is only used if the change affects our MongoDB Realm API's and we normally link to the issue if there is one. Otherwise 👍